docs/example.rst

```diff
@@ -19,39 +16,33 @@ Example
       -[channel] *120*456#
 
 
-   Favourite Colour
-   ~~~~~~~~~~~~~~~~
+   ### Favourite Colour
```

Comment:
I'm personally a big fan of underlining -- it allows my eye to find section start and ends easily which the `#`'s don't.

We can always support both later if its enough of an issue though, so fine with this change.